doc/lsst.source.injection/reference/41_match_injected_outputs.rst

```diff
@@ -0,0 +1,194 @@
+-------------------------------------------------
+
+This page covers how to match injected input catalogs to output data catalogs.
+This process can generally be split into two parts: consoliding per-patch injected catalogs into tract-level input catalogs, and matching the input and output catalogs.
+
+Consolidate injected catalogs
+=============================
+
+The butler may split up catalogs which cover multiple photometry bands or which cover large areas of sky for memory efficiency, even when a single catalog is injected.
+For example, if a coadd-level injected catalog covers a whole tract across multiple photometry bands, the injected catalogs will be split and stored with the dimensions ``{patch, band}``.
+Before matching the injected input catalogs to the processed output catalog, the per-patch and per-band inputs must be consolidated into a single tract level catalog.
+This can be done by using ``pipetask run`` to run ``ConsolidateInjectedCatalogsTask`` from ``pipelines/consolidate_injected_deepCoadd_catalogs.yaml``.
+
+.. code::
+
+    pipetask --long-log --log-file $LOGFILE run \
+    -b $REPO \
+    -i $PROCESSED_DATA_COLL \
+    -o $CONSOLIDATED_CATALOG_COLL \
+    -p $SOURCE_INJECTION_DIR/pipelines/consolidate_injected_deepCoadd_catalogs.yaml \
+    -d "instrument='HSC' AND skymap='hsc_rings_v1' AND tract=9813 AND patch=42 AND band='i'"
+
+*where*
+
+    `$LOGFILE`
+        The full path to a user-defined output log file.
+
+    `$REPO`
+        The path to the butler repository.
+
+    `$PROCESSED_DATA_COLL`
+        The name of the input injected catalog collection.
+
+    `$CONSOLIDATED_CATALOG_COLL`
+        The name of the consolidated injected output collection.
+
+
+Matching
+========
+
+Now that we have our consolidated tract-level injected catalog and a reference tract-level standard catalog, we can move on to matching these two sets of catalogs together.
+.. TODO: add info for match_object_to_truth and compare_object_to_truth
+The matching tasks are ``MatchTractCatalogTask`` and ``DiffMatchedTractCatalogTask``.
+The first task performs a spatial probablistic match with minimal flag cuts, and the second computes any relevant statistics.
+These tasks are located under ``pipelines/match_injected_tract_catalog.yaml#match_object_to_truth,compare_object_to_truth`` alongside ``ConsolidateInjectedCatalogsTask``.
+The pipeline graph for the consolidation and matching process is shown below:
+      ○  injected_deepCoadd_catalog
+      │
+    ○ │  skyMap
+    ├─┤
+    │ ■  consolidate_injected_catalogs
+    │ │
+    │ ○  injected_deepCoadd_catalog_tract
+    │ │
+  ○ │ │  injected_objectTable_tract
+╭─┼─┼─┤
+■ │ │ │  match_object_to_truth
+│ │ │ │
+◍ │ │ │  match_target_injected_deepCoadd_catalog_tract_injected_objectTable_tract, match_ref_injected_deepCoadd_catalog_tract_injected_objectTable_tract
+╰─┴─┴─┤
+      ■  compare_object_to_truth
+      │
+      ○  matched_injected_deepCoadd_catalog_tract_injected_objectTable_tract
```

Comment:
Have you tried building the docs to see how this looks? I suspect you'll need to wrap this in a code block to make it appear as monospace (and hence keep the correct spacing for all of the nodes and edges).

Reply:
Don't forget a newline before and after the code block as well.